wiki/People/The_Team/Beatmap_Nominators/General_Information/fr.md

```diff
@@ -1,61 +1,57 @@
+- Le [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895) : Un outil créé par [Naxess](https://osu.ppy.sh/users/8129817), un membre de la NAT. Cet outil permet de vérifier un large éventail de problèmes empêchant une beatmap d'être classée, et une fonctionnalité de capture instantanée pour facilement comparer les versions d'une map, ce qui est utile lors des revérifications. **Cela ne doit pas remplacer la vérification d'une beatmap par vous même**, mais plutôt être utilisé comme un outil de double-vérification au cas où vous auriez loupé quelque chose.
 
 ## Les capacités des Beatmap Nominators
 
-Les nominations et les resets de nominations démontrent les efforts d'un BN lors de la vérification des maps, indiquant la progression d'une map à travers le processus de classement. En tant que BN, vous avez accès à ces outils et vous devez les utiliser à bon escient.
+Les nominations et les resets de nominations démontrent les efforts d'un BN lors de la vérification des beatmaps, indiquant la progression d'une beatmap à travers le processus de classement. En tant que BN, vous avez accès à ces outils et vous devez les utiliser à bon escient.
 
 ### Nominer
 
 ![Bouton de nomination](img/nominate-fr.png)
 
-**Ce bouton vous permet de nominer des maps.** Pour nominer une beatmap, cette dernière doit avoir au moins 5 [Hypes](/wiki/Beatmap/Hype) et aucun problème(s) ou de suggestion(s) non résolu. Assurez vous que vous êtes satisfait de toutes les difficultés, de même pour les autres modes de jeu, avant d'utiliser ce bouton. Si la beatmap a fait l'objet d'un veto par le passé, vous devez faire attention à ce que le veto soit retiré avant de nominer une beatmap. Quand une beatmap possède 2 nominations, elle sera [qualifiée](/wiki/Beatmap/Category#qualifiée) et entrera dans la file d'attente pour le classement.
+**Ce bouton vous permet de nominer des beatmaps.** Pour nominer une beatmap, cette dernière doit avoir au moins 5 [Hypes](/wiki/Beatmap/Hype) et ne doit pas avoir de problèmes ou de suggestions non résolus. Assurez-vous que vous êtes satisfait de toutes les difficultés, de même pour les autres modes de jeu, avant d'utiliser ce bouton. Si la beatmap a fait l'objet d'un veto par le passé, vous devez faire attention à ce que le veto soit retiré avant de nominer une beatmap. Quand une beatmap possède 2 nominations, elle sera [qualifiée](/wiki/Beatmap/Category#qualifiée) et entrera dans la file d'attente pour entrer dans la catégorie classée.
 
 ### Disqualifier ou réinitialiser la nomination
 
 ![Bouton de disqualification](img/reset_d-fr.png)
 ![Bouton de réinitialisation de la nomination](img/reset_q-fr.png)
```

Comment:
not caused by this PR but this really should be a single image like EN, so the figure/caption thing can display correctly